notNeededPackages.json

```diff
@@ -3573,6 +3573,10 @@
             "libraryName": "lower-case-first",
             "asOfVersion": "1.0.1"
         },
+        "lowlight": {
+            "libraryName": "lowlight",
+            "asOfVersion": "2.0.0"
```

Comment:
I think this should be 2.0.1. Even though the release notes state that types were added for 2.0.0, that release seems broken, because the `index.d.ts` file references `./lib/common.js`, which wasn't published in version 2.0.0, but is present in 2.0.1.

Reply:
Done.